catroid/src/org/catrobat/catroid/devices/mindstorms/ev3/EV3Motor.java

```diff
@@ -36,16 +36,16 @@ public EV3Motor(int port) {
 		//this.port = port;
 
 		switch (port) {
-			case 0:
+			case 0: // A
```

Comment:
If you can't use enums for some reasons, consider to use named constants combined with the enum annotation.
https://developer.android.com/tools/debugging/annotations.html#enum-annotations
https://developer.android.com/reference/android/support/annotation/IntDef.html